docs/noise.md

```diff
@@ -0,0 +1,121 @@
+## Noise
+
+For simulation it is useful to have `Gate`s that enact noisy quantum evolution.
+Currently Cirq supports modeling noise via *operator sum* representations of
+noise (these evolutions are also known as quantum operations or quantum
+dynamical maps).  This formalism models evolution of the density matrix
+via
+
+![Operator sum representation: $\rho \rightarrow \sum_k A_k \rho A_k^\dagger$](resources/OperatorSumDef.gif)
+
+Where here the the A<sub>k</sub> are *Krauss* operators. These operators
+are not necessarily unitary and must satisfy the trace preserving
+property
+
+![Operator sum normalization: $\sum_k A_k^\dagger A_k = I$](resources/OperatorSumNormDef.gif)
```

Comment:
Do these show correctly on the readthedocs site?

Reply:
I assume so because this is the same pattern used for circuits.html, for example.  Not sure if there is an easy way to test this?

Reply:
Do they appear in the HTML when the docs are built locally?

Reply:
yes.  they get moved to _images and that shows up when you build locally.